test/sql/collate/test_collate_and_grouping_sets.test

```diff
@@ -0,0 +1,78 @@
+# name: test/sql/collate/test_collate_and_grouping_sets.test
+# description: Test collation and grouping sets.
+# group: [collate]
+
+require icu
+
+statement ok
+set default_collation=c;
+
+statement ok
+CREATE TABLE sales (
+   product_id INT,
+   region VARCHAR(50),
+   year INT,
+   amount_sold DECIMAL(10,2)
+);
+
+statement ok
+INSERT INTO sales VALUES
+  (1, 'North', 2020, 1000.00),
+  (1, 'North', 2021, 1500.00),
+  (1, 'South', 2020, 800.00),
+  (1, 'South', 2021, 700.00),
+  (2, 'North', 2020, 500.00),
+  (2, 'North', 2021, 600.00),
+  (2, 'South', 2020, 400.00),
+  (2, 'South', 2021, 550.00);
+
+statement ok
+set default_collation=c;
+
+statement ok
+create table t1 as SELECT product_id, region, SUM(amount_sold) AS total_amount
+FROM sales
+GROUP BY GROUPING SETS ((product_id), (region), ())
+ORDER BY product_id, region, total_amount;
+
+statement ok
+set default_collation=en_us;
+
+statement ok
+create table t2 as SELECT product_id, region, SUM(amount_sold) AS total_amount
+FROM sales
+GROUP BY GROUPING SETS ((product_id), (region), ())
+ORDER BY product_id, region, total_amount;
+
+query I
+select count(*) from (select * from t1 INTERSECT select * From t2);
+----
+5
+
+mode skip
```

Comment:
Is this left in on purpose?

Reply:
Ah, no, just forgot to take it out. Need to clean up this test too actually